src/ucp/wireup/wireup.c

```diff
@@ -679,22 +679,23 @@ static UCS_F_NOINLINE void
 ucp_wireup_send_ep_removed(ucp_worker_h worker, const ucp_wireup_msg_t *msg,
                            const ucp_unpacked_address_t *remote_address)
 {
+    unsigned ep_init_flags = UCP_EP_INIT_ERR_MODE_PEER_FAILURE;
```

Comment:
is this needed because of https://github.com/openucx/ucx/pull/6401/files#diff-1a6cd9df9fc4c2ac2f3e662ca4ce0ff54e0ad405bab393e80090c8593745351cR2342?